about.html

```diff
@@ -0,0 +1,118 @@
+                Our World has changed. - Hack your way to the top
+            </h3>
+            <p class="title-description">
+                WEB3 is here to help reimagine your world. Write, test and deploy scalable decentralized applications on the most developer-friendly blockchain using the languages you already know – even Javascript.
+            </p>
+            <span class="contact-us">
+                Please contact us per Email for any further questions about WEB3 Hackathon 2022!
+            </span>
+            <p class="contact-email">
+                web3hackathon2022@techblaze.org
+            </p>
+        </main>
+    </div>
+    </section>
+    <section class="hackathon-logo" id="hackathon-logo">
+        <h2 class="hackathon-logo-title">WEB3 Hackathon 2022 Logo</h2>
+        <hr>
+        <p class="logo-design">
+            The logo for the 2022 WEB3 HACKATHON was decided through a unanimous community vote after 
+            different designs have been submitted by talented designers in the community. The design from
+            Valentine Blaze emerged the winner, accumulating the highest votes from the pulll.
+        </p>
+        <div class="hackathon-logo-img">
+            <img src="./assets/images/Web3-Hackathon.png" alt="Hackathon logo" class="hackathon-logo-image">
+        </div>
+    </section>
+    <section  id="past-events">
+        <div class="past-event-section">
+            <h2 class="past-event-title">See the past Web3 Hackathon Events</h2>
+            <hr>
+            <p class="past-info">
+                Take a look at the last two Web3 Hackathon events which took place in Lagos-Nigeria and London
+            </p>
+            <div class="past-event-images">
+                <div class="event-1">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past Event 1"> -->
+                        <span class="event-year"> 2021</span>
+                        <span class="event-detail"> Web3 Hackathon 2021 Lagos-Nigeria</span>
+                    </div>
+                </div>
+                <div class="event-2">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past event 2"> -->
+                        <span class="event-year"> 2021</span>
+                        <span class="event-detail"> Web3 Hackathon 2021 London</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="partner-footer-mobile">
+        <h2 class="partner-title">
+            Partners
+        </h2>
+        <hr>
+        <div class="partner-logos">
+            <img src="./assets/images/FintechSurge-Logo_new.png" alt="FintechSurge Logo">
+            <img src="./assets/images/Gitex-logo-2.png" alt="Gitex logo">
+            <img src="./assets/images/NORTH-STAR-DUBAI-LOGO-New-2.png" alt="NORTH STAR DUBAI LOGO New">
+        </div>
+
+    </section>
```

Comment:
- [ ] Kindly try to follow design guidelines as much as possible. For the partners section the background color should be `#414246` and without `border-radius` for the top corners 👍 
## Your design:
![image](https://user-images.githubusercontent.com/50721479/196670798-7b942a76-c1c6-4216-8622-825e88d3b848.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/196670970-52acb220-c08e-4785-825d-1d94d0cff51d.png)